src/azure-cli-testsdk/azure/cli/testsdk/__init__.py

```diff
@@ -15,6 +15,6 @@
 __all__ = ['ScenarioTest', 'LiveTest', 'ResourceGroupPreparer', 'StorageAccountPreparer',
            'RoleBasedServicePrincipalPreparer', 'CliTestError', 'JMESPathCheck', 'JMESPathCheckExists', 'NoneCheck',
            'live_only', 'record_only', 'StringCheck', 'StringContainCheck', 'get_sha1_hash', 'KeyVaultPreparer',
-           'JMESPathCheckGreaterThan', 'api_version_constraint']
+           'JMESPathCheckGreaterThan', 'api_version_constraint', 'execute']
```

Comment:
Hm, at some point maybe I got the impression that something used it, but I think I was mistaken. I'll remove that.